recipes-kernel/linux/linux-toradex/SPI-CAN.cfg

```diff
@@ -0,0 +1,14 @@
+CONFIG_SPI=y
+CONFIG_SPI_FSL_LPSPI=y
+CONFIG_SPI_FSL_QUADSPI=y
+CONFIG_SPI_NXP_FLEXSPI=y
+CONFIG_SPI_IMX=y
+CONFIG_SPI_FSL_DSPI=y
+CONFIG_SPI_PL022=y
+CONFIG_SPI_ROCKCHIP=y
+CONFIG_SPI_SPIDEV=y
+CONFIG_SPI_SLAVE=y
+CONFIG_SPI_SLAVE_TIME=y
+CONFIG_SPI_SLAVE_SYSTEM_CONTROL=y
+CONFIG_CAN=m
```

Comment:
What does `m` mean?

Reply:
that means a loadable module, so prolly have to run modprobe before using it.  btw file would change since SPI is already enabled. I would have to probably get rid of all the SPI related stuff I added here..

Reply:
Thanks.